viewer/viewer-awt/src/main/java/org/icepdf/ri/common/SwingControllerFactory.java

```diff
@@ -0,0 +1,27 @@
+package org.icepdf.ri.common;
+
+import org.icepdf.ri.common.views.Controller;
+
+import java.util.ResourceBundle;
+
+/**
+ * A factory of SwingController
+ */
+public final class SwingControllerFactory implements ControllerFactory {
+
+    public static final SwingControllerFactory INSTANCE = new SwingControllerFactory();
```

Comment:
For API consistency can you convert this to a getInstance() method and make the constructor private.